repotracker/repotracker.go

```diff
@@ -732,11 +733,13 @@ func createVersionItems(v *model.Version, ref *model.ProjectRef, metadata Versio
 		var buildId string
 		buildId, err = model.CreateBuildFromVersion(args)
 		if err != nil {
-			grip.Error(message.WrapError(err, message.Fields{
-				"message": "error inserting build",
-				"version": v.Id,
-				"variant": buildvariant.DisplayName,
-			}))
+			if metadata.TriggerID == "" || !mgo.IsDup(err) {
```

Comment:
What's the purpose of checking that trigger ID is empty here?

Reply:
This is to avoid logging duplicate key errors for triggered builds